Adding ShipmentMode fix in the changelog (#1059)
mercadopago/devsite-docs: guides/resources/changelog/current-year.en.md

## February 2021

### February 16th

> CHANGELOG
>

> > Fixed serialization of the ShipmentMode enum in the .NET SDK

Reviewer: ```suggestion
> Changes to ShipmentMode Enum Serialization
```

>

> > NEWS: NEWS

Reviewer: ```suggestion
> FIX: FIX
```

>
> PRODUCT: SDK .NET

> We have fixed ShipmentMode enum serialization to generate values accepted by the Preference API

Reviewer: ```suggestion
We have fixed the `ShipmentMode` enum serialization in the .Net C # SDK to generate values accepted by the Preferences API.
```
